### PR TITLE
Attempt to address different showmount path in centos/ubuntu for jenkins

### PR DESCRIPTION
**Pull request by ajdecon**
Our CentOS nightly run in Jenkins is failing on `test-slurm-nfs-mount.sh` because it can't find `showmount`:

```
debug1: Sending command: showmount -e | grep home
bash: showmount: command not found
```

I suspect the issue here is that `showmount` isn't in the default PATH for this SSH command. Unfortunately, Ubuntu and CentOS put `showmount` in different locations...

This change to the test adds a conditional for the CentOS case and swaps to use the absolute path `/usr/sbin/showmount` in the CentOS case.

Unfortunately this only shows up in our nightly Jenkins test so this is difficult to test 😢

### CODE REVIEW

**Comment by ajdecon** [comment]
Please don't merge until we confirm this still passes the per-PR test on Ubuntu.

**Review by supertetelman — APPROVED** [review]
LGTM, let's merge through after `deepops-nightly/681/` completes.

**Pull request merged**
